FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -633,22 +635,23 @@ def export_getComponentDocumentation( self, cType, system, module ):
     except Exception as _e:
       return S_ERROR( 'No documentation was found' )
 
-  types_storeHostInfo = []
-  def export_storeHostInfo( self ):
+  @staticmethod
+  def __storeHostInfo():
     """
     Retrieves and stores into a MySQL database information about the host
     """
-    result = self.__readHostInfo()
+    result = SystemAdministratorHandler.__readHostInfo()
     if not result[ 'OK' ]:
       gLogger.error( result[ 'Message' ] )
-      return S_ERROR( result[ 'Message' ] )
+      return result
 
     fields = result[ 'Value' ]
     fields[ 'Timestamp' ] = datetime.utcnow()
+    fields[ 'Extension' ] = fields[ 'Extensions' ]
```

Comment:
This is needed just because the DB column name is wrong. But in the patch we are cautious with changing the DB schema. in v6r16 the column name should be changed and this fix removed, and an appropriate release note added